pipeline/translate/decoder.yml

```diff
@@ -1,10 +1,17 @@
 normalize: 1.0
 word-penalty: 0
-mini-batch: 16
-mini-batch-words: 500
+# doesn't seem to affect GPU utilization
+#max-length-factor: 2
+
+mini-batch: 512
+#mini-batch-words: 2000
 maxi-batch: 1000
 maxi-batch-sort: src
 max-length: 200
 max-length-crop: true
 beam-size: 8
-quiet-translation: True
\ No newline at end of file
+quiet-translation: true
+#precision: float16
```

Comment:
we use this one for faster encoding. Should we just add another decoder config for opus models and add a config option for this?